packages/atomic/src/pages/examples/ipx.html

```diff
@@ -189,12 +189,32 @@
             <atomic-query-error></atomic-query-error>
             <atomic-no-results></atomic-no-results>
           </atomic-layout-section>
-        </div>
-        <div slot="footer">
           <atomic-layout-section section="pagination">
-            <atomic-pager></atomic-pager>
+            <atomic-load-more-results style="padding-top: 0.875rem"></atomic-load-more-results>
           </atomic-layout-section>
         </div>
+        <div style="display: flex; justify-content: space-between; align-items: center" slot="footer">
+          <style>
+            .ipx-footer-link {
+              color: var(--atomic-primary);
+              text-decoration: none;
+            }
+
+            .ipx-footer-icon {
+              aspect-ratio: auto;
+              width: 5rem;
+              height: 1.25rem;
+            }
+          </style>
```

Comment:
notsure. this is a publi repo, is it smart to mention coveo@coveo there?

Reply:
**Ideally** it would work anywhere yes, not just for internal folks, but it can always be done later... I don't think the presence of Coveo@Coveo is necessarily a mistake, depends on your requirements for IPX

Reply:
It's just an example of what we want for clients. It can be replaced by something else.